data/virt_autotest/host_autoyast/autoyast_opensuse_kvm_sshd.xml

```diff
@@ -0,0 +1,143 @@
+      <activate>true</activate>
+      <boot_mbr>true</boot_mbr>
+      <gfxmode>auto</gfxmode>
+      <os_prober>true</os_prober>
+      <terminal>console serial</terminal>
+      <serial>serial --unit=1 --speed=115200 --word=8 --parity=no</serial>
+      <timeout t="integer">30</timeout>
+      <update_nvram>true</update_nvram>
+      <xen_kernel_append>console=com2,115200 log_lvl=all guest_loglvl=all sync_console</xen_kernel_append>
+      <xen_append>console=hvc0,115200 console=tty loglevel=5</xen_append>
+    </global>
+  </bootloader>
+  <partitioning t="list">
+    <drive>
+      <device>/dev/sda</device>
+      <initialize t="boolean">true</initialize>
+      <use>all</use>
+      <enable_snapshots t="boolean">false</enable_snapshots>
+    </drive>
+  </partitioning>
+  <timezone>
+    <hwclock>UTC</hwclock>
+    <timezone>America/New_York</timezone>
+  </timezone>
+  <networking>
+    <keep_install_network t="boolean">true</keep_install_network>
+    <backend>network_manager</backend>
+  </networking>
+  <software>
+    <products t="list">
+      <product>openSUSE</product>
+    </products>
+    <install_recommended t="boolean">true</install_recommended>
+    <patterns t="list">
+      <pattern>base</pattern>
+      <pattern>basesystem</pattern>
+      <pattern>enhanced_base</pattern>
+      <pattern>kvm_server</pattern>
+      <pattern>kvm_tools</pattern>
+    </patterns>
+    <packages t="list">
+      <package>autoyast2</package>
+      <package>supportutils</package>
+      <package>nmap</package>
+    </packages>
+  </software>
+  <users t="list">
+    <user>
+      <username>bernhard</username>
+      <user_password>$y$j9T$xsY9ubdI0jRF5JoSV8vvg1$03I5jIewyiCMm/EmkgRtYxvEYm83ZSJo0tl1rKP8Ad.</user_password>
+      <fullname>Bernhard M. Wiedemann</fullname>  
+      <encrypted t="boolean">true</encrypted>
+    </user>
+    <user>
+      <username>root</username>
+      <user_password>$y$j9T$Bc3oHTMRcmYxyzTlxEd7A/$jU0KChUccTna2tH9ItMRyRHC9fofibG37RGH4XkCxYD</user_password>
+      <encrypted t="boolean">true</encrypted>
+    </user>
+  </users>
+  <services-manager>
+    <default_target>multi-user</default_target>
+    <services>
+      <disable t="list">
+        <service>firewalld</service>
+      </disable>
+      <enable t="list">
+        <service>sshd</service>
+	<service>virtqemud</service>
+	<service>virtstoraged</service>
+	<service>virtnetworkd</service>
+	<service>virtnodedevd</service>
+      </enable>
+    </services>
+  </services-manager>
+  <scripts>
+    <!-- permit root login, password login and pubkeys login -->
+    <!-- keep ssh session alive -->
+    <!-- /etc/ssh/sshd_config is not default installed in TW -->
+    <init-scripts t="list">
+      <script>
+        <source><![CDATA[
+sshd_config_file="/etc/ssh/sshd_config"
+if [ ! -f $sshd_config_file ]; then
+    mkdir -p `dirname $sshd_config_file`
+    echo -e "PermitRootLogin yes\nPubkeyAuthentication yes\nPasswordAuthentication yes" > $sshd_config_file
```

Comment:
If I remember correctly, the sshd config file for tumbleweed is ```/usr/share/etc/ssh/sshd_config``` ?

Reply:
There is no `/usr/share/etc/ssh/sshd_config` at present. I guess you are saying `/usr/etc/ssh/sshd_config`.  

```
~ # cat /usr/etc/ssh/sshd_config
# To modify the system-wide sshd configuration, create a "*.conf" file under
# "/etc/ssh/sshd_config.d/" which will be automatically included below.
# Don't edit this configuration file itself if possible to avoid update
# problems.
```

I once tried with creating a .conf file under "/etc/ssh/sshd_config.d/", but it did not take effects. "/etc/ssh/sshd_config" was found to be effective. Don't know why, but I will keey an eye on it.

Reply:
There is no `/usr/share/etc/ssh/sshd_config` at present. I guess you are saying `/usr/etc/ssh/sshd_config`.  

```
~ # cat /usr/etc/ssh/sshd_config
# To modify the system-wide sshd configuration, create a "*.conf" file under
# "/etc/ssh/sshd_config.d/" which will be automatically included below.
# Don't edit this configuration file itself if possible to avoid update
# problems.
```

I once tried with creating a .conf file under "/etc/ssh/sshd_config.d/", but it did not take effects. "/etc/ssh/sshd_config" was found to be effective. Don't know why, but I will keey an eye on it.

Reply:
Just tried, adding two *.conf files under "/etc/ssh/sshd_config.d/" works, permit_root_login.conf and keep_alive.conf. so "/etc/ssh/sshd_config" will not be used in TW anymore.

Reply:
Just tried, adding two *.conf files under "/etc/ssh/sshd_config.d/" works, permit_root_login.conf and keep_alive.conf. so "/etc/ssh/sshd_config" will not be used in TW anymore.